packages/MSBot/docs/update-service.md

```diff
@@ -0,0 +1,41 @@
+## Updating Service
+To update a service with new information you can use the msbot update set of commands
+
+```shell
+msbot update [serviceType] --[requiredArg] [value]
+```
+
+This allows you to rename a service, update keys, etc.
+
```

Comment:
I can get this list from -help. It's better to add actual sample usage here.

Reply:
Yes. but the docs list it so you can look at the scope of functionality offered by the tool even if you don't yet have the tool installed.